data/redirects.yaml

```diff
@@ -25,6 +25,8 @@
 "/rest/beta#batch": "/rest/batch"
 "/rest/channels-messages": "/rest/channels"
 "/rest/versions/v0.8/channels-messages": "/rest/versions/v0.8/channels"
+"/how-ably-works": "/root/key-concepts"
+"/root/how-ably-works": "/root/key-concepts"
```

Comment:
the 'root' doesn't appear in the final URL right?

Reply:
It doesn't - I remove it at the web end. Just added the direct to cover the base of someone hitting it.

Reply:
It doesn't - I remove it at the web end. Just added the direct to cover the base of someone hitting it.